$OpenBSD: patch-src_calcurse_h,v 1.1 2018/03/22 23:15:29 fcambus Exp $

Fix date display in calendar, the %-* format string is a GNU libc
extension which is not supported by strftime(3) on OpenBSD.

Index: src/calcurse.h
--- src/calcurse.h.orig
+++ src/calcurse.h
@@ -302,7 +302,7 @@ enum datefmt {
     (datefmt == DATEFMT_YYYYMMDD ? _("yyyy/mm/dd") : _("yyyy-mm-dd"))))
 
 /* Day heading default format. */
-#define DAY_HEADING_DEFAULT "%B %-d, %Y"
+#define DAY_HEADING_DEFAULT "%B %d, %Y"
 
 struct date {
